src/views/Proposals.vue

```diff
@@ -28,21 +28,28 @@
     <Container :slim="true">
       <Block :slim="true">
         <div class="px-4 py-3 bg-gray-dark overflow-auto menu-tabs">
-          <a
+          <router-link
+            :key="'core'"
+            v-text="'core'"
+            :to="'/'+key+'/core'"
```

Comment:
We want to use template strings everywhere https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
```suggestion
            :to="`${key}/core`"
```